playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -8,7 +8,7 @@
     openshift_deployment_type: "{{ deployment_type }}"
     registry_image: "{{  openshift.master.registry_url | replace( '${component}', 'docker-registry' )  | replace ( '${version}', openshift_image_tag ) }}"
     router_image: "{{ openshift.master.registry_url | replace( '${component}', 'haproxy-router' ) | replace ( '${version}', openshift_image_tag ) }}"
-    registry_console_image: "{{ openshift.master.registry_url | replace ( '${component}', 'registry-console') | replace ( '${version}', 'v' ~ openshift.common.short_version ) }}"
+    registry_console_image: "{{ openshift.master.registry_url | replace ( 'ose-${component}', 'registry-console') | replace ( '${version}', 'v' ~ openshift.common.short_version ) }}"
```

Comment:
It seems it doesn't and `regex_replace` isn't picking it up..
Any qualms with something like 
```
registry_console_image: "{{ openshift.master.registry_url | replace('origin-', 'ose-') | replace ( 'ose-${component}', 'registry-console') | replace ( '${version}', 'v' ~ openshift.common.short_version ) }}"
```